include/tc/core/cuda/cuda_execution_engine.h

```diff
@@ -128,18 +129,6 @@ class CudaExecutionEngine {
       const std::string& name,
       const std::vector<const DLTensor*>& inputsInfo,
       const MappingOptions& options);
-  size_t emplaceExecutor(std::unique_ptr<ExecutorInfo> p);
-
-  /// For thread-safety perform all cheap operations under lock
-  std::mutex executorInfoMutex;
-
-  // XXX:if ExecutorInfo is moved/copied (even when the vector's underlying
-  // storage is extended) something inside isl segfaults,  unique_ptr is used as
-  // a workaround
```

Comment:
This comment got dropped.
If it is no longer relevant, then it should be dropped separately.